x-pack/legacy/plugins/spaces/public/copy_saved_objects_to_space/_index.scss

```diff
@@ -1 +1 @@
-@import './components/index';
\ No newline at end of file
+@import './components/copy_to_space';
```

Comment:
We're closer 😆 

I think we still need the `_index.scss` in `/components which imports copy_to_space`. Then in this parent folder, import said index file.